common/src/java/com/zimbra/common/service/ServiceException.java

```diff
@@ -303,6 +303,10 @@ public static ServiceException LICENSE_ERROR(String message, Throwable cause) {
         return new ServiceException("license error: "+message, LICENSE_ERROR, RECEIVERS_FAULT, cause);
     }
 
+    public static ServiceException ERROR_MESSAGE(String str, Throwable cause){
+        return new ServiceException("error: " + str, null, SENDERS_FAULT, cause);
```

Comment:
Avoid using `+` operator for String concatenation, consider using `String.format` or `formatted` if upgraded to Java 15 or above.